scipy/optimize/_differentialevolution.py

```diff
@@ -97,12 +97,13 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         denoted by CR. Increasing this value allows a larger number of mutants
         to progress into the next generation, but at the risk of population
         stability.
-    seed : int or `np.random.RandomState`, optional
-        If `seed` is not specified the `np.RandomState` singleton is used.
-        If `seed` is an int, a new `np.random.RandomState` instance is used,
+    seed : {int, `~np.random.RandomState`, `~np.random.Generator`}, optional
+        If `seed` is not specified the `~np.random.RandomState` singleton is
+        used.
+        If `seed` is an int, a new ``RandomState`` instance is used,
```

Comment:
same question as above, my guess is the answer is the same for all-just calling out to note this occurs in multiple spots in PR.

Reply:
It's more important here that the docstring changes are uniform.

Reply:
right between this and Robert's comment above I think we can call these comments "not an issue". Thanks.